packages/fx-core/src/question/scaffold/vsc/CapabilityOptions.ts

```diff
@@ -0,0 +1,587 @@
+    return {
+      id: "api-spec",
+      label: getLocalizedString(
+        "core.createProjectQuestion.capability.copilotPluginApiSpecOption.label"
+      ),
+      detail: getLocalizedString(
+        "core.createProjectQuestion.capability.messageExtensionApiSpecOption.detail"
+      ),
+      data: TemplateNames.ApiPluginExistingApi,
+    };
+  }
+}
+
+export class OfficeAddinCapabilityOptions {
+  static outlookAddinImport(): OptionItem {
+    return {
+      id: "outlook-addin-import",
+      label: getLocalizedString("core.importAddin.label"),
+      detail: getLocalizedString("core.importAddin.detail"),
+      data: TemplateNames.OfficeAddinCommon,
+    };
+  }
+  static officeContentAddin(): OptionItem {
+    return {
+      id: "office-content-addin",
+      label: getLocalizedString("core.officeContentAddin.label"),
+      detail: getLocalizedString("core.officeContentAddin.detail"),
+    };
+  }
+  static officeAddinImport(): OptionItem {
+    return {
+      id: "office-addin-import",
+      label: getLocalizedString("core.importOfficeAddin.label"),
+      detail: getLocalizedString("core.importAddin.detail"),
+      description: getLocalizedString(
+        "core.createProjectQuestion.option.description.previewOnWindow"
+      ),
+      data: TemplateNames.OfficeAddinCommon,
+    };
+  }
+  static outlookTaskPane(): OptionItem {
+    return {
+      id: "outlook-json-taskpane",
+      label: getLocalizedString("core.newTaskpaneAddin.label"),
+      detail: getLocalizedString("core.newTaskpaneAddin.detail"),
+      data: TemplateNames.OutlookTaskpane,
+    };
+  }
+  static wxpTaskPane(): OptionItem {
+    return {
+      id: "wxp-json-taskpane",
+      label: getLocalizedString("core.newTaskpaneAddin.label"),
+      detail: getLocalizedString("core.newTaskpaneAddin.detail"),
+      data: TemplateNames.WXPTaskpane,
+    };
+  }
+}
+
+export class BotCapabilityOptions {
+  static readonly basicBotId = "bot";
+  static readonly notificationBotId = "notification";
+  static readonly commandBotId = "command-bot";
+  static readonly workflowBotId = "workflow-bot";
+
+  static basicBot(): OptionItem {
+    return {
+      id: BotCapabilityOptions.basicBotId,
+      label: `${getLocalizedString("core.BotNewUIOption.label")}`,
+      detail: getLocalizedString("core.BotNewUIOption.detail"),
+      data: TemplateNames.DefaultBot,
+    };
+  }
+  static aiBot(): OptionItem {
+    return {
+      id: "ai-bot",
+      label: getLocalizedString("core.aiBotOption.label"),
+      detail: getLocalizedString("core.aiBotOption.detail"),
+      data: TemplateNames.AIBot,
+    };
+  }
+
+  static aiAssistantBot(): OptionItem {
+    return {
+      id: "ai-assistant-bot",
+      label: getLocalizedString("core.aiAssistantBotOption.label"),
+      detail: getLocalizedString("core.aiAssistantBotOption.detail"),
+      description: getLocalizedString("core.createProjectQuestion.option.description.preview"),
+      data: TemplateNames.AIAssistantBot,
```

Comment:
It's confusing here because this template(several others too) is only used in VS question model, should be split later instead of placing under vsc folder.